### PR TITLE
tests merged into one end to end test

### PR DESCRIPTION
**Pull request by xdragu01**
Before:
```
============================================================================================================ test session starts =============================================================================================================
platform linux -- Python 3.7.4, pytest-4.6.3, py-1.8.0, pluggy-0.12.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/pdragun/Documents/robottelo
plugins: forked-1.0.2, xdist-1.29.0, mock-1.10.4, cov-2.7.1, services-1.3.1
collecting ... 2019-08-22 08:38:02 - conftest - DEBUG - Collected 26 test cases

collected 26 items                                                                                                                                                                                                                           

tests/foreman/cli/test_host_collection.py ...s.................s....                                                                                                                                                                   [100%]
============================================================================================ 24 passed, 2 skipped, 1 warnings in 1760.22 seconds =============================================================================================

```

After:
```
============================================================================================================ test session starts =============================================================================================================
platform linux -- Python 3.7.4, pytest-4.6.3, py-1.8.0, pluggy-0.12.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/pdragun/Documents/robottelo
plugins: forked-1.0.2, xdist-1.29.0, mock-1.10.4, cov-2.7.1, services-1.3.1
collecting ... 2019-08-26 11:48:29 - conftest - DEBUG - Collected 10 test cases

collected 10 items                                                                                                                                                                                                                           

tests/foreman/cli/test_host_collection.py .s......s.                                                                                                                                                                                   [100%]
============================================================================================= 8 passed, 2 skipped, 1 warnings in 600.90 seconds ==============================================================================================
```

### CODE REVIEW

File: tests/foreman/cli/test_host_collection.py [hunk 1]
```diff
@@ -679,35 +330,6 @@ def test_positive_copy_by_id(self):
         })
         self.assertEqual(result['name'], new_name)
 
-    @tier2
-    def test_positive_copy_by_name(self):
```
Comment (ntkathole) [inline tests/foreman/cli/test_host_collection.py]: Can you please confirm if we are not loosing coverage for host collection copy?

File: tests/foreman/cli/test_host_collection.py [hunk 2]
```diff
@@ -64,43 +65,99 @@ def _make_fake_host_helper(self):
             u'organization-id': self.organization['id'],
         })
 
+    @upgrade
     @tier1
```
Comment (vijay8451) [inline tests/foreman/cli/test_host_collection.py]: I think more related tier is now tier2/3 ? could you please check.

File: tests/foreman/cli/test_host_collection.py (hunk 1 above)
Comment (xdragu01) [reply tests/foreman/cli/test_host_collection.py]: There is another test that copy host collections by id few lines above. So I think this one is not necessary.

File: tests/foreman/cli/test_host_collection.py (hunk 2 above)
Comment (xdragu01) [reply tests/foreman/cli/test_host_collection.py]: Good point. Updated

**Pull request merged**